doc/Makefile

```diff
@@ -7,7 +7,7 @@
 # Use explicit "version_info" indexing since make cannot handle colon characters, and
 # evaluate it now to allow easier debugging when printing the varaible
 
-PYVER:=$(shell python3.7 -c 'from sys import version_info as v; print("{0}.{1}".format(v[0], v[1]))')
+PYVER:=$(shell python -c 'from sys import version_info as v; print("{0}.{1}".format(v[0], v[1]))')
```

Comment:
The issue that prompted the change in gh-10708 was that both python3.6 and python3.7 were available, but only python3.7 was set up with the SciPy development environment. The original command
```
PYVER:=$(shell python3 -c 'from sys import version_info as v; print("{0}.{1}".format(v[0], v[1]))')
```
returned `3.6`, so `make html` would fail.

Will this fix that? (If so, will it still work if python3.6 has the development environment and python3.7 doesn't?)

Reply:
That's not an issue that can be dealt with that way. If you have multiple Python's installed, you should be using conda envs or virtualenvs to activate the Python you want.

Reply:
@mdhaber, @andyfaff you can also use [`update-alternatives`](https://linux.die.net/man/8/update-alternatives) to register `python3.7` as the default for `python3`.